libs/cdk/src/lib/utils/directives/selectable-list/selectable-item.directive.ts

```diff
@@ -81,7 +81,7 @@ export class SelectableItemDirective<ElementType extends Element = HTMLElement,
         private disabled$: FdkDisabledProvider,
         private readonly$: FdkReadonlyProvider,
         private selectionService: SelectionService<ElementType, ValueType>,
-        private _elementRef: ElementRef<ElementType>,
+        public readonly elementRef: ElementRef<ElementType>,
```

Comment:
The same comment as above. just missing reason / connection why we change this private / public modifier - comment would be helpful

Reply:
Hey @fkolar. Thank you for your review. The reason why this part got changes is that main interface `HasElementRef` was changed (it was used somethere for table directives/components), and that affected majority of other classes. The reason to move from method call to a direct property access is to reduce amount of function evaluations (even though it had only `return this._elementRef` line, VM would still need to evaluate it), and to simplify the components itself.